### PR TITLE
feat: Expose important library versions

### LINKED ISSUE #3415: Expose versions of important underlying libraries
`type_defect`   |    by rjsparks@nostrum.com

___

It will be useful for people who are coordinating the behavior of other tools with the datatracker to know which versions of things like xml2rfc, markdown, and similar are currently in use.

Consider adding them to the release api, or creating a new one.

See the comments at https://mailarchive.ietf.org/arch/msg/rfc-interest/qMkpn-JdtU7E6s26sCOSZeVYT3I/

___
_Issue migrated from trac:3415 at 2022-03-04 09:14:58 +0000_

### PR DESCRIPTION
**Pull request by kesara**
Update `/api/version` to include

```
"other": {
    "xml2rfc": "<version>",
    "weasyprint": "<version>"
},
```

Fixes #3415

### CODE REVIEW

**Comment by kesara** [comment]
@rjsparks is this implementation going in the correct direction?

**Comment by rjsparks** [comment]
Yes - these seems reasonable.

**Comment by rjsparks** [comment]
@kesara - do you want to continue to pursue this?

**Comment by kesara** [comment]
> @kesara - do you want to continue to pursue this?

Yes. I will.

File: ietf/settings.py [hunk 1]
```diff
@@ -1279,6 +1279,8 @@ def skip_unreadable_post(record):
 
 PUBLISH_IPR_STATES = ['posted', 'removed', 'removed_objfalse']
 
+IMPORTANT_LIBRARIES = ["django", "celery", "lxml", "markdown", "psycopg2", "pyang", "rfc2html", "weasyprint", "xml2rfc"]
```
Comment (rjsparks) [inline ietf/settings.py:1282]: Lets have a group conversation this week about the content of this list.

Comment (rjsparks) [reply ietf/settings.py:1282]: Missed this week - aim for next - added a card

**Review by jennifer-richards — DISMISSED** [review]
lgtm

File: ietf/settings.py [hunk 2]
```diff
@@ -1279,6 +1279,8 @@ def skip_unreadable_post(record):
 
 PUBLISH_IPR_STATES = ['posted', 'removed', 'removed_objfalse']
 
+IMPORTANT_LIBRARIES = ["markdown", "pyang", "rfc2html", "xml2rfc"]
```
[the remark this answers is not in the document]
Comment (kesara) [reply ietf/settings.py]: Renamed with https://github.com/ietf-tools/datatracker/pull/7713/commits/4d917fb93e4e78341af816d28672c92eda3fc8a3

**Review by rjsparks — APPROVED** [review]

**Pull request merged**